render/query.go

```diff
@@ -128,7 +128,7 @@ func FetchDataPoints(ctx context.Context, cfg *config.Config, fetchRequests Mult
 		if tf.MaxDataPoints <= 0 {
 			tf.MaxDataPoints = int64(cfg.ClickHouse.MaxDataPoints)
 		}
-		targets.pointsTable, targets.isReverse, targets.rollupObj = SelectDataTable(cfg, tf.From, tf.Until, targets.List, chContext)
+		targets.pointsTable, targets.isReverse, targets.rollupUseReverted, targets.rollupObj = SelectDataTable(cfg, tf.From, tf.Until, targets.List, chContext)
```

Comment:
I already feel like it makes sense to return `*config.DataTable, error` from SelectDataTable. And then change `render.Targets` accordingly. Does it look reasonable?

Reply:
Yes, like targets.isReverse. It's per table config option. So, returned from SelectDataTable.

Reply:
Also reorg SelectDataTable (as method of Targets struct), return error only. All required fileds set in method directly.